activate.ps1

```diff
@@ -37,7 +37,7 @@ deactivate -init
 
 $_OLD_PATH = $env:PATH
 # Tell dotnet where to find itself
-$env:DOTNET_ROOT = "$PSScriptRoot\.dotnet\x64"
+$env:DOTNET_ROOT = "$PSScriptRoot\.dotnet"
```

Comment:
Does this mean we don't have the x86 version of dotnet anymore?

Reply:
We do, but its in .dotnet\x86\.

Which reminds me, I'm going to update this PR to also set the `DOTNET_ROOT(x86)` env var from these scripts